assets/js/main.js

```diff
@@ -3,6 +3,36 @@ $(function(){
     $('.sidenav, .overlay').toggleClass('is-visible');
     e.preventDefault();
   });
+
+  var generateCodeSnippets = function(content, previewBox) {
+
+    var self = this;
+
+    this.parseCode = function(previewBox) {
```

Comment:
I don't understand the point of having theses functions as instance members on the generateCodeSnippets function if we're not using that as a class. Why wouldn't this just be:

```js
function parseCode(previewBox) {

}
```

Same goes for render.